### PR TITLE
New scispacy feedstock:

### PR DESCRIPTION
**Pull request by akabanovs**
- Upstream: https://github.com/allenai/scispacy/tree/v0.5.2

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+test:
+  imports:
+    - scispacy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://spacy.io/universe/project/scispacy
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:44]: maybe that would be better https://allenai.github.io/scispacy/ ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  run:
+    - python 
+    - spacy >=3.4.0,<3.5.0
+    - requests >=2.0.0,<3.0.0
+    - conllu
+    - numpy
+    - joblib
+    - nmslib >=1.7.3.6
+    - scikit-learn >=0.20.3
+    - pysbd
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:33]: does it run without it ?, it says in upstream is for testing or  "custom sentence splitter" which I really dunno if it used or not

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
if you think my comments are not relevant then, LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml:44]: I don't have strong opinion on this one.
The page you have mentioned can be reached from the url I added by clicking 'view more'. I would stick with the url currently in the recipe.

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml:33]: Not sure. They mention it as a requirement in setup.py:
https://github.com/allenai/scispacy/blob/v0.5.2/setup.py#L51

Comment (akabanovs) [reply recipe/meta.yaml:33]: Looking at the code it seems it uses it during runtime. Also, this scispacy is about processing text, so I expect this pysbd package to be used.

**Pull request merged**